test/comprehensive/edl/deepcopy.edl

```diff
@@ -70,6 +82,8 @@ enclave {
 
     // TODO: We should have a `SizeStruct` to test deep copying where
     // the size attribute is correctly used with a hard-coded value.
+    public void deepcopy_size([in, out, count=1] SizeStruct* s);
```

Comment:
Can we remove the TODO above? The test takes care of the scenario in the TODO.

Reply:
Sure, Thanks